assets/src/components/FeaturesTable.js

```diff
@@ -82,8 +91,22 @@ export default class FeaturesTable extends HTMLElement {
             fields += ',' + this.sortingField;
         }
 
+        let uniqueFieldJSON = 'null';
+
+        // Create a unique JSON object for PHP request
+        if (!this.isFieldJSONEmpty()) {
+            uniqueFieldJSON = '{';
+
+            this.fieldJSON.field.forEach(field => {
+                uniqueFieldJSON += `"${field.alias}": "${field.expression}",`;
+            });
+
+            uniqueFieldJSON = uniqueFieldJSON.slice(0, -1);
+            uniqueFieldJSON += '}';
+        }
```

Comment:
Please rename also and dot not create a string but a JavaScript object which will be transformed into JSON in the `getFeatures` method